stdlib/src/builtin/string.mojo

```diff
@@ -135,6 +133,74 @@ fn chr(c: Int) -> String:
     return String(p.bitcast[DType.int8](), num_bytes + 1)
 
 
+# ===----------------------------------------------------------------------===#
+# ascii
+# ===----------------------------------------------------------------------===#
+
+
+@always_inline("nodebug")
+fn chr_ascii(c: Int8) -> String:
```

Comment:
About this and the other ascii helpers: Could you please prepend them with `_`? At this point, I'm not sure where these will live, and which of them will survive, so I would rather not make them public.

Reply:
sure